### DISCUSSION TITLE
The properties panel does not appear in the "floating window" and "search"

### DISCUSSION THREAD

**Discussion by cordinGH** [Macavity/siyuan-database-properties-panel#72] [Ideas]
This plugin is great. Thank you so much for developing it.
But there may be a bit of a problem:
The current properties panel can only be displayed when the document is opened.
But it can't be displayed in "search" or "floating windows".

**Comment by Macavity** [comment]
Yes, those views are not part of the document window and this plugin won't try to change them. Unless you want to propose to do this yourself?

**Comment by cordinGH** [comment]
> Yes, those views are not part of the document window and this plugin won't try to change them. Unless you want to propose to do this yourself?

Thank you for clarifying! I'm not familiar with development, so I might not be able to implement this feature myself. Would it be possible to consider this as a future enhancement for the plugin?

**Comment by Macavity** [comment]
Ah these days with ChatGPT anyone can be a developer :) 

Sure, I think it makes sense to keep this around as a potential enhancement. But I don't have endless amounts of time on my hands and I don't personally see a lot of priority for this 🤷‍♂️ 
So, if this is important to you, you might be worth to consider doing it yourself.

**Comment by Macavity** [comment]
Support for floating windows was already added a while ago, but now it's also properly working for search results:
<img width="1728" height="996" alt="SearchResults" src="https://github.com/user-attachments/assets/b2f94697-d6c4-4da7-ae7a-a3befe4f7d6a" />
